app/controllers/api/v1/teachers_controller.rb

```diff
@@ -0,0 +1,33 @@
+module Api
+  module V1
+    class TeachersController < ApplicationController
```

Comment:
I thought I saw this with different syntax `Api::V1::TeachersController < ApplicationController` should bring this down to one line.

Reply:
This is an update after the Rubocop updates

Reply:
This was an auto-correct from Rubocop